<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="16347" started-at="2019-12-31T17:12:25Z" finished-at="2019-12-31T17:12:41Z">
    <groups>
    </groups>
    <test name="HomePage Test" duration-ms="8198" started-at="2019-12-31T17:12:25Z" finished-at="2019-12-31T17:12:33Z">
      <class name="SportsEngine.HomePage">
        <test-method status="PASS" signature="basePageNavigationForward_01()[pri:0, instance:SportsEngine.HomePage@272ed83b]" name="basePageNavigationForward_01" duration-ms="6991" started-at="2019-12-31T17:12:25Z" finished-at="2019-12-31T17:12:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basePageNavigationForward_01 -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:SportsEngine.HomePage@272ed83b]" name="teardown" is-config="true" duration-ms="1158" started-at="2019-12-31T17:12:32Z" finished-at="2019-12-31T17:12:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- SportsEngine.HomePage -->
    </test> <!-- HomePage Test -->
    <test name="ValidateText Test" duration-ms="7809" started-at="2019-12-31T17:12:34Z" finished-at="2019-12-31T17:12:41Z">
      <class name="SportsEngine.ValidateText">
        <test-method status="PASS" signature="setupTest()[pri:0, instance:SportsEngine.ValidateText@d29f28]" name="setupTest" is-config="true" duration-ms="6086" started-at="2019-12-31T17:12:34Z" finished-at="2019-12-31T17:12:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method status="PASS" signature="validateMessageFindSportTx_02()[pri:0, instance:SportsEngine.ValidateText@d29f28]" name="validateMessageFindSportTx_02" duration-ms="66" started-at="2019-12-31T17:12:40Z" finished-at="2019-12-31T17:12:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateMessageFindSportTx_02 -->
        <test-method status="FAIL" signature="validateMessageGetOut_01()[pri:0, instance:SportsEngine.ValidateText@d29f28]" name="validateMessageGetOut_01" duration-ms="42" started-at="2019-12-31T17:12:40Z" finished-at="2019-12-31T17:12:40Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA['Get out', Header Text Incorrect expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 'Get out', Header Text Incorrect expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at SportsEngine.ValidateText.validateMessageGetOut_01(ValidateText.java:30)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateMessageGetOut_01 -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:SportsEngine.ValidateText@d29f28]" name="teardown" is-config="true" duration-ms="1120" started-at="2019-12-31T17:12:40Z" finished-at="2019-12-31T17:12:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- SportsEngine.ValidateText -->
    </test> <!-- ValidateText Test -->
  </suite> <!-- Suite -->
</testng-results>
